.github/workflows/sync-issue-with-pr.yaml

```
@@ -35,6 +35,8 @@ jobs:
           fi
 
           echo "Linked issues: $ISSUE_NUMBERS"
+          
```

yamllint

[error] trailing spaces

(trailing-spaces)